libycp/src/Import.cc

```diff
@@ -108,6 +108,15 @@ Import::import (const string &name, Y2Namespace *preloaded_namespace)
 	    }
 	}
 
+        // for runtime error during loading lets act like if module is there, just error namespace is created
+        if (isErrorNamespace(name_space))
+        {
+            ycp2error("Runtime error when loading module '%s'", cname);
+            m_active_modules.insert (std::make_pair (m_name, name_space));			// insert to list of known modules
+	    m_module = m_active_modules.find (m_name);
+            return 0;
```

Comment:
The existing comments are quite confusing, but I guess what matters most is the first condition in [`import(name, preloaded_namespace)`](https://github.com/yast/yast-core/blob/0c7b73439de267a21486366b495b5f5d97206f7d/libycp/src/Import.cc#L66): it means it will retry loading the module if the name is empty, that is, if the method has previously returned -1.

So we should add a documenting comment to this method
> // return 0 if done (successfully or not), -1 if we should retry